Formula/biogeme.rb

```diff
@@ -0,0 +1,39 @@
+class Biogeme < Formula
+  desc "Maximum likelihood estimation of choice models"
+  homepage "http://biogeme.epfl.ch"
+  url "http://biogeme.epfl.ch/distrib/biogeme-2.5.tar.gz"
+  sha256 "88548e99f4f83c24bf7ddb8e0de07588adc2bec515569c56e816ed5b20a624b3"
+
+  option "with-bison", "Install also Bison Biogeme"
```

Comment:
So… is there a reason to install it? :D 
We try to limit the number of options provided by a formula to its minimum. I’m just checking if this option is necessary.